docs/units/standard_units.rst

```diff
@@ -162,7 +162,26 @@ For example::
    Unit(dimensionless with a scale of 1000.0)
    >>> (u.km / u.m).decompose() == u.dimensionless_unscaled
    False
-
+   
+To create a dimensionless quantity with a specific scaling factor, 
+multiply or divide a value by ``Unit()`` with the scaling 
+factor specified, e.g.::
+
+   >>> big_unitless_num = 20000000 # 20 million
+
+   >>> # I will be doing many calculations with big_unitless_num, 
+   >>> # but I want all of my answers to be multiples of a million:
+   >>> big_unitless_num /= 1e6
+
+   >>> # The above will work, but I want to keep track of this scaling
+   >>> # factor through all of my calculations. u.Unit() can do this
+   >>> # for us:
+   >>> big_unitless_num *= u.Unit(1e6)
```

Comment:
I think this is clearer as a new definition of the number, i.e.,
```
big_unitless_num = 20 * u.Unit(1e6)
```
Partly, this is because the in-place operation would not work well if the number was an array instead (long-standing bug, difficult to fix...).